src/services/db-manager.ts

```diff
@@ -83,6 +83,7 @@ export function grantPostgresGroupRoleQuery(user: string, dbId: string) {
   const groupRole = getGroupRole(dbId);
   return `
     GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO ${groupRole};
+    REVOKE INSERT, UPDATE, DELETE ON iasql_audit_log FROM ${groupRole};
```

Comment:
Perhaps it would make sense to have this in the `iasql_platform` module instead of here?

Reply:
where exactly? does not need to be here per user basis? since we are granting everything for all tables in public? Or should we move the audit log to another schema?

Reply:
I think so, all those tables should be out of bounds